gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode,
+  NxP,
+  NxTable,
+  NxTile,
+  NxH2,
+  NxH3,
+  NxTextLink,
+  NxStatefulAccordion,
+  NxAccordion,
+  NxWarningAlert
+} from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { GalleryTileFooter } from '../../gallery-components/GalleryTileFooter';
+
+import CodeExample from '../../CodeExample';
+
+const NxToastComplexLayoutExampleCode = require('./NxToastComplexLayoutExample?raw'),
+    NxToastSimpleLayoutExampleCode = require('./NxToastSimpleLayoutExample?raw'),
+    NxToastLegacyLayoutExampleCode = require('./NxToastLegacyLayoutExample?raw');
+
+const NxToastPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Toasts are rendered on the right side the screen, underneath the header, and will remain in place until the user
+        dismisses them individually. Each toast renders one
+        {' '}<NxCode><NxTextLink href="#/pages/Alert">NxAlert</NxTextLink></NxCode>, which comes in four
+        variations: Error, Info, Warning, and Success.
+      </NxP>
+      <NxP>
+        Toasts generally fall under a "global and static" context, in a sense that throughout the application,
+        toasts are rendered within the same container on right side of the screen.
+      </NxP>
+      <NxWarningAlert>
+        Note: In order to render toasts in the correct position, all toasts must be wrapped within the parent React
+        component <NxCode>NxToastContainer</NxCode>, which must be a direct child of <NxCode>.nx-page</NxCode>.
+      </NxWarningAlert>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>NxToastContainer</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Prop</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>children</NxTable.Cell>
+              <NxTable.Cell>React Node</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                All <NxCode>NxToast</NxCode>s rendered within <NxCode>NxToastContainer</NxCode>.
+                <NxWarningAlert>
+                  Note: To ensure proper focusing order, <NxCode>NxToast</NxCode>s must be rendered in descending order,
```

Comment:
It's not really about focus order.  It's that we intend more recent toasts to render at the top of the toast container, which is the start of the `NxToastContainer`'s children.  It is up to the calling code to ensure that.

Reply:
> It's not really about focus order. It's that we intend more recent toasts to render at the top of the toast container, which is the start of the `NxToastContainer`'s children. It is up to the calling code to ensure that.

Docs changed here: [af9654f](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/af9654f1872db8cd57f0b6d11babe640d721aeb9)